src/views/Dashboards/Dashboards.js

```diff
@@ -35,6 +35,7 @@ function a11yProps (index) {
 const Dashboards = ({ currentScenario }) => {
   const classes = useStyles();
   const [value, setValue] = useState(0);
+  const noScenario = currentScenario === null;
```

Comment:
`noScenario` seems to be declared as a bool (not sure, without parenthesis tho)...

Reply:
Done